isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -87,6 +93,27 @@
       </p>
       </blockquote>
 
+      <blockquote>
+      <b>Solving for the target body radii (triaxial or mean) is irrelavent and
+      likely increases error in the solve.</b>
+      <p>
+        When solving for target body radii both the mean and triaxial solve options
+        are redundantly solving for the target body radii. The radii solve is part of
```

Comment:
This is not so much redundant as not possible. There is no dependence of the back projection upon the target radii. It instead depends upon the local point radii which can then be collectively evaluated, as described later, to fit target radii.

Reply:
Be very careful about terminology here. Target radii == the radii of an ellipsoid fit to the body, while local radii == the local radius of each control point. The two are not the same, but they are closely related. Each point radius is a local approximation of the global target radii.

Reply:
Reworded and added to this section to mention that the local point radii is being solved for rather than just some magic radii